Sources/Commands/SwiftTool.swift

```diff
@@ -605,6 +605,13 @@ public class SwiftTool {
             // User didn't tell us to use these .netrc files so be more lenient with errors
             func loadNetrcNoThrows(at path: AbsolutePath) -> NetrcAuthorizationProvider? {
                 guard localFileSystem.exists(path) else { return nil }
+                
+                do {
+                    try withTemporaryFile(dir: path.parentDirectory) { _ in }
+                } catch {
+                    self.observabilityScope.emit(warning: "\(path.parentDirectory) is not accessible or not writable, not using .netrc file in it. \(error)")
```

Comment:
Would just use a colon here.

```suggestion
                    self.observabilityScope.emit(warning: "\(path.parentDirectory) is not accessible or not writable, not using .netrc file in it: \(error)")
```